src/main/java/cst438flights/controller/FlightController.java

```diff
@@ -0,0 +1,30 @@
+package cst438flights.controller;
+
+import cst438flights.service.FlightService;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Controller;
+import org.springframework.ui.Model;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestParam;
+
+@Controller
+public class FlightController {
+
+    @Autowired
+    private FlightService flightService;
+
+    @PostMapping("/flight/reservation")
+    public String createReservation(
+            @RequestParam("departureAirport") String departureAirport,
+            @RequestParam("arrivalAirport") String arrivalAirport,
+            @RequestParam("departureDate") String departureDate,
+            Model model) {
+
```

Comment:
This will get expanded with my pull request for all of the reservation requirements